### PR TITLE
Fix Virtual Wan Point to site feature resources property names from swagger. 

### PR DESCRIPTION
**Pull request by Nilambari**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by Nilambari** [comment]
This is to get bug fix from August branch:- https://github.com/Azure/azure-rest-api-specs/pull/7498

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/P2SVpnGatewayGet.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "parameters": {
     "gatewayName": "p2sVpnGateway1",
     "resourceGroupName": "rg1",
-    "api-version": "2019-09-01",
+    "api-version": "2019-08-01",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/P2SVpnGatewayGet.json]: Wrong API version (in this and other examples)

**Review by anton-evseev — CHANGES_REQUESTED** [review]

**Comment by lirenhe** [comment]
@Nilambari  please fix the code check errors in NetworkValidation and modelValidation.

**Comment by Nilambari** [comment]
> @Nilambari  please fix the code check errors in NetworkValidation and modelValidation.

Done.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/P2SVpnGatewayGet.json (hunk 1 above)
Comment (Nilambari) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/P2SVpnGatewayGet.json]: Corrected.

**Comment by Nilambari** [comment]
Can you Please help approve PR?

**Review by anton-evseev — APPROVED** [review]

**Comment by number213** [comment]
@lirenhe please take a look

**Review by lirenhe — APPROVED** [review]

**Pull request merged**